packages/@aws-cdk/aws-stepfunctions-tasks/README.md

```diff
@@ -370,6 +370,43 @@ const task = new tasks.CodeBuildStartBuild(this, 'Task', {
 });
 ```
 
+### StopBuild
+
+[StopBuild](https://docs.aws.amazon.com/codebuild/latest/APIReference/API_StopBuild.html) attempts to stop running a build.
+
+```ts
+import * as tasks from '@aws-cdk/aws-stepfunctions-tasks';
```

Comment:
i think these import statements might get in the way of translating these snippets to other languages.
try running `../../../scripts/compile-code-snippets`

don't worry if there are other straggler failures, let's just keep these ones in good shape :)